src/hatch/env/collectors/plugin/interface.py

```diff
@@ -1,5 +1,9 @@
 from __future__ import annotations
 
+from typing import Any, Dict
```

Comment:
With `from __future__ import annotations` please use built-in types

```suggestion
from typing import Any
```

Reply:
I updated the annotations where relevant and added the future imports in [`238b62f` (#520)](https://github.com/pypa/hatch/pull/520/commits/238b62fd3239a055500cf180fa325ea2305f7be3) and [`9b03298` (#520)](https://github.com/pypa/hatch/pull/520/commits/9b03298fd1f7dad2e63bcd434464d7b26df0646c)